src/pykeen/models/inductive/inductive_nodepiece.py

```diff
@@ -163,14 +164,19 @@ def __init__(
             self.num_valid_entities = validation_factory.num_entities
             self.num_test_entities = test_factory.num_entities
 
-    def _entity_representation_from_mode(self, *, mode: Optional[InductiveMode]):
-        assert mode is not None, "Inductive mode must be explicitly set for inductive models"
+    # docstr-coverage: inherited
+    def _get_entity_representations_from_inductive_mode(
+        self, *, mode: Optional[InductiveMode]
+    ) -> Sequence[Representation]:  # noqa: D102
+        if mode is None:
+            raise ValueError(f"{self.__class__.__name__} does not support inductive mode: {mode}")
```

Comment:
this would always print None - is it by design or you wanted to check for some unexpected mode string values?

Reply:
It has been there before, as
```python
assert mode is not None, "Inductive mode must be explicitly set for inductive models"
```
so I guess it had some intention 😄

Reply:
see also https://github.com/pykeen/pykeen/pull/929#discussion_r876799931